ecologits/impacts/llm.py

```diff
@@ -2,8 +2,9 @@
 from math import ceil
 from typing import Any, Optional, Union, cast
 
+from ecologits._ecologits import EcoLogits
```

Comment:
This module should not depend on `_ecologits.py` or we risk creating circular dependencies. This module should use low-level only parameters only. Meaning it is not aware of electricity mixes per location, the list of providers and models, etc. Only the raw formulas.